src/appengine/libs/mail.py

```diff
@@ -33,10 +36,10 @@ def send(to_email, subject, html_content):
     return
 
   message = Mail(
-      from_email=str(from_email),
-      to_emails=str(to_email),
+      from_email=From(str(from_email)),
+      to_emails=To(str(to_email)),
       subject=subject,
```

Comment:
Do we not need to do anything with subject ?

Reply:
Good catch. This worked without it because all of the instances of newstr we actually used came from the explicit str conversions here, but if we passed one in we'd need to use sendgrid's Subject. Will fix this in another change.